jina/executors/compound.py

```diff
@@ -210,12 +210,12 @@ def is_updated(self) -> bool:
         return (self.components and any(c.is_updated for c in self.components)) or self._is_updated
 
     @is_updated.setter
-    def is_updated(self, val: bool):
+    def is_updated(self, val: bool) -> bool:
```

Comment:
this is not true, it is a setter so it does not have any return value, it returns `None`

Reply:
Done